using-directus-as-a-baby-health-tracker/index.md

```diff
@@ -0,0 +1,403 @@
+---
+title: 'Using Directus as a Baby Health Tracker'
+description: 'This article highlights using Directus as a central backend for smart wearables, offering step-by-step guides for setup, data retrieval, and integration with third-party tools like OpsGenie and iPhone widgets.'
+author:
+  name: 'Andreas Morgner'
+  avatar_file_name: 'Andreas_Morgner.jpg'
+---
+
+## Introduction
+Directus can be used for many different use cases all over certain industries. Next to more comment use cases like a backend for websites, asset management or simple data management from an existing database, this article is about to use of Directus as a health-tracking tool for a baby or toddler. 
+
+## Summary
+In this article, we have a look at how Directus can be used as a central backend for any (or at least the most smart) wearable device and how to extend the functionality of the the apps shipped with the gadget.
+You will find some basic step-by-step guides for setting up some Collections, relationships between Collections, how to automatically grab data from the wearable using Flows and how to integrate Directus into 3rd party tools for reporting, such as OpsGenie and screen widgets on an iPhone.
+
+## Before You Start
+
+### Setup and Technical requirements
+To get on common ground, let's have a look at the used tools and infrastructure first.
+Next to a running Directus instance, regardless if self-hosted or within Directus Cloud. 
+For Directus I expect you to have the permission to create new Collections as well as new Flows, so an admin-access level would be required. 
+
+Next, you need a wearable with an API to consume the data. Of course, all other sensor data would work as long as we can access the data by a web request. Depending on your used sensors, the final Flow might look different. In our case, we used the Neebo tracker first and switched to Owlet. Both provide an API with some authentication set up in front. Unfortunately, there is no official documentation of the API so it was kind of reverse engineering the network traffic and digging through a couple of GitHub repositories.
+
+Last but not least, a basic understanding of Directus and RESTful APIs and web requests is helpful, but we'll provide as many details as possible to make it as easy as possible to follow, even if you're new to Directus (or APIs).
+## Implementation
+### Collections
+
+> [!TIP] What is a Collection
+> Collections are individual collections of items, similar to tables in a database. Changes to collections will alter the schema of the database. [Learn more about Collections](https://docs.directus.io/user-guide/overview/glossary#collections).
+
+First, let's point out which Collections we need to save the data later on. To make it simple, we start with only two Collections. From a data-structure point of view, this might not be the best solution, but as Directus is very flexible, we can optimize the structure later on if needed.
+#### The data storage collection
```

Comment:
Remove all this. You don't need to teach them what a collection is, and you don't have to weigh up the pros/cons of your data model. Just remain direct and instructional.